test/cases/compile_errors/incorrect_type_to_memset_memcpy.zig

```diff
@@ -27,10 +40,14 @@ pub export fn non_matching_lengths() void {
 // :5:5: error: unknown @memcpy length
 // :5:18: note: destination type '[*]u8' provides no length
 // :5:24: note: source type '[*]align(4) const u8' provides no length
-// :10:13: error: type 'u8' does not support indexing
-// :10:13: note: operand must be an array, slice, tuple, or vector
-// :15:13: error: type '*u8' does not support indexing
-// :15:13: note: operand must be an array, slice, tuple, or vector
+// :10:13: error: type '*u8' is not an indexable pointer
+// :10:13: note: operand must be a slice, a many pointer or a pointer to an array
```

Comment:
Destination may be a tuple, previous message was more correct

Reply:
My understanding was that that hasn't been implemented yet #15479?

Reply:
That's correct, it's not implemented yet. It's fine to do as you have it, as long as you're aware :+1:

Reply:
There is a link to the issue next to the error message as a reminder.